.github/workflows/flake8.yml

```diff
@@ -0,0 +1,66 @@
+name: Python code quality check
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # This workflow contains two job called "flake8-grass7-scripts" and "flake8-gui-wxgui-scripts"
```

Comment:
```suggestion
```

I can see that! :-)

Reply:
Still it may be helpful to newcomers :)

Reply:
Let me invoke the rule of writing comments in the source code. Don't comment on the obvious things. We don't do:

```python
# list which contains 1 and 2
a = [1, 2]
```

Surely this would be helpful to people who has newer seen Python before, but a simple search on Python syntax reveals what `a = [1, 2]` is. Same for:
```yaml
jobs:
  flake8-grass7-scripts:
    ...
  flake8-grass7-gui:
    ...
```

I'm far from advocating for minimal comments or doc (see, e.g., #20). However, things which should be documented are intention and non-obvious syntax or behavior. This is YAML, not regular expressions, Bash, or [Whitespace](https://en.wikipedia.org/wiki/Whitespace_(programming_language)).

If you want to document overall intention of this script, why the strategy, etc, that would be something different.

Reply:
alright!